activity_log.php

```diff
@@ -30,9 +30,6 @@
 $sys = ( $is_admin && getGetValue ( 'system' ) != '' );
 
 print_header();
-
-ob_start();
```

Comment:
I put them in back when I only had a dial up modem. Sending pages in bigger chunks was WAY more efficient. However, I'm taking most of them out now.
Also, there used to be places where characters were sent and then headers/cookies were tried to send. ob_start() sorted those out. I think that problem has been resolved now, too.